scripts/server.dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM golang:1.23 AS builder
+FROM --platform=$BUILDPLATFORM golang:latest AS builder
```

Comment:
Is it a best practice to pin the dependency version for production / release artifacts? Should we just use golang:1.24 here?

I'm fine with :latest for the test images.

Reply:
Only now, and only for Go :). Because we declare the toolchain, it will always build with the correct Go version.